airflow/providers/amazon/aws/hooks/lambda_function.py

```diff
@@ -106,6 +106,9 @@ def create_lambda(
         image_config: Any | None = None,
         code_signing_config_arn: str | None = None,
         architectures: list[str] | None = None,
+        ephemeral_storage: Any | None = None,
```

Comment:
Can this be Any? Or should we have a proper type and define a default as well?

Reply:
Hi @amoghrajesh , Basically the added parameters are dict as per boto3 configuration, Here there are already few dict parameters defined as Any, I have followed the similar approach for these new parameters. 

And the default is set to None because there is call to trim the none values, Before creating function.

Reply:
I see, thank you!